core/CronArchive.php

```diff
@@ -751,6 +751,9 @@ public function invalidateArchivedReportsForSitesThatNeedToBeArchivedAgain($idSi
             return;
         }
 
+        $this->logger->debug("Applying queued rearchiving...");
+        $this->invalidator->applyScheduledReArchiving();
```

Comment:
@diosmosis could we maybe call this somewhere else so it's called less often? Eg only at the start of the cron archive which be typically every hour?

Reply:
That should only be called once per core:archive.

Reply:
not sure I'm seeing it right. Isn't it called as part of `getNextArchivesToProcess()` and therefore as part of the while loop?

Reply:
Oh you're right, nevermind, I had it at the start of run() initially but moved it for some reason. I'll move it bacck.